### PR TITLE
QX deprecated, runout and shellout

### PR DESCRIPTION
**Pull request by AlexDaniel**
Description of this pull request was moved to https://github.com/perl6/specs/pull/102.

### CODE REVIEW

**Comment by pmichaud** [comment]
FWIW, this looks to me like a request to change the language (Perl 6) with an attached pull request to modify the compiler (Rakudo).  

I think the discussion of the language issues properly belong somewhere else, such as perl6-language or as a language issue ticket (https://github.com/perl6/specs).  Then, once there's agreement on the language change to be made, it makes sense to look at this pull request.

In short, we've gone to a lot of trouble to keep the compiler (Rakudo) design and tickets separate from the language (Perl 6) design and tickets, and we should aim to keep it that way.

Pm

**Comment by AlexDaniel** [comment]
@pmichaud I agree! I've submitted a pull request there. However, I expected to see a comment about the proposed change…

**Comment by AlexDaniel** [comment]
https://github.com/perl6/specs/pull/102 is now closed, so I am closing this as well.
